includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -499,7 +499,7 @@ public function cancel_authorization( WP_REST_Request $request ) {
 
 			$result = $this->gateway->cancel_authorization( $order );
 
-			if ( Intent_Status::SUCCEEDED !== $result['status'] ) {
+			if ( Intent_Status::CANCELED !== $result['status'] ) {
```

Comment:
Changelog file is missing

Reply:
Added here: https://github.com/Automattic/woocommerce-payments/pull/7150/commits/3b2f6e0da41695585f33ae8cd715699e7969eaa9